UNRELEASED.md

```diff
@@ -13,6 +13,8 @@
 
 ### Documentation
 
+- Improved split button example ([#2791](https://github.com/Shopify/polaris-react/pull/2791))
```

Comment:
I don’t think we need this added to the change log since it’s not a change to a component. You can just use the skip change log label in this pr to get that check to pass.